Modules/LearningSequence/classes/LSItems/ilLSItemsDB.php

```diff
@@ -22,12 +22,20 @@ class ilLSItemsDB
 	 */
 	protected $container_sorting;
 
+	/**
+	 * @var ilLSPostConditionDB
+	 */
+	protected $post_conditions;
```

Comment:
kannst du die variable noch in $post_conditions_db umbenennen? ist sonst nicht eindeutig.